src/pages/Product/Components/ProductList/ProductList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ProductItem from './ProductItem';
+import './ProductList.scss';
+
+const ProductList = ({ productData }) => {
+  return (
+    <ul className="productList">
+      {productData.data.map(productList => {
```

Comment:
productData, productList 등에서 이것이 어떤 정보를 표현하는지 명확히 이해하기 힘듭니다. 실제로 List가 아니라 하나의 product가 각 매개변수로 나오는 것 같고요

```suggestion
      {products.map(product => {
```

위 같은 형태가 되는게 가장 깔끔해보이네요!

Reply:
수정하였습니다! 앞으로 map의 매개변수도 이름을 잘 지어야겠다는 생각이 듭니다.

Reply:
반영이 안되있습니다.

Reply:
착각하고 다른곳에 수정해버렸습니다. 다시 재 수정하였습니다.